### ISSUE TITLE
Devicelab window1 causes infra failures

### ISSUE THREAD

**Issue by keyonghan** [flutter/flutter#114930]
This bot has been failing tests consistently. Manually quarantine the bot to unblock the tree.

<img width="1277" alt="Screen Shot 2022-11-08 at 5 35 37 PM" src="https://user-images.githubusercontent.com/54558023/200715198-e09babb4-c4d9-4cf1-b52d-a4a1cf9f3f12.png">

Example build: https://ci.chromium.org/ui/p/flutter/builders/prod/Windows_android%20basic_material_app_win__compile/7120/overview

```
Cannot run a document in the middle of a pipeline: C:\salt\Get-ChildItem.
At line:1 char:1
+ gci -Recurse cert: |Where-Object {$_.Subject -like "*GTS CA 1D4*" -or ...
+ ~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (C:\salt\Get-ChildItem:String) [], RuntimeException
    + FullyQualifiedErrorId : CantActivateDocumentInPipeline
```

**Comment by danagbemava-nc** [comment]
Closing this in favor of https://github.com/flutter/flutter/issues/114935
